src/Blocks/BlockSnow.h

```diff
@@ -11,6 +11,8 @@ class cBlockSnowHandler :
 	public cBlockHandler
 {
 public:
+	enum { FullBlockMeta = 7 };
```

Comment:
Wrap this into multiple lines, add a comment to the number `// Meta value of a full-height snow block`